daffodil-lib/src/test/scala/UpdateEclipseClasspaths.scala

```diff
@@ -93,7 +104,17 @@ trait UpdateEclipseClasspaths {
     val srcs = srcsDir.listFiles().flatMap { _.listFiles().flatMap { _.listFiles() } }
     val docs = docsDir.listFiles().flatMap { _.listFiles().flatMap { _.listFiles() } }
 
-    val triples = (bundles ++ jars).map { jar =>
+    //
+    // Remove duplicates between the bundles directory and jars directory
+    // (Not sure why sbt populates both, but many (all?) things found under
+    // the bundles directory are also found under jars.
+    //
+    // Prefer the jars directory if things are found in two places.
+    //
```

Comment:
I have since learned that sbt compile populates the bundles, and sbt updateClassifiers populates the duplicate jars. Not sure why,